CorsixTH/Lua/entities/humanoids/staff.lua

```diff
@@ -670,19 +670,21 @@ function Staff:getDrawingLayer()
   return 4
 end
 
---! Estimate staff service quality based on skills, fatigue and happiness.
+--! Estimate staff service quality based on skills, restfulness (inverse of fatigue) and happiness.
 --!return (float) between [0-1] indicating quality of the service.
 function Staff:getServiceQuality()
   -- weights
   local skill_weight = 0.7
-  local fatigue_weight = 0.2
+  local restfulness_weight = 0.2
   local happiness_weight = 0.1
 
   local weighted_skill = skill_weight * self.profile.skill
-  local weighted_fatigue = fatigue_weight * self.attributes["fatigue"]
+  -- Less fatigue is better
+  local weighted_restfulness = restfulness_weight * -self.attributes["fatigue"]
```

Comment:
You've introduced a caveat here where service quality can no longer go above 0.8. Instead consider this. 
```suggestion
  local weighted_restfulness = 0.2 + (restfulness_weight * -self.attributes["fatigue"])
```
Or to be more logical make it so the + is the -